docs/source/examples.rst

```diff
@@ -0,0 +1,15 @@
+Examples
+========
+
+There are several examples ported from `pytorch/examples <https://github.com/pytorch/examples>`_ using `ignite`
+to display how it helps to write compact and full-featured training loops in a few lines of code:
+
+- Mnist
```

Comment:
maybe add individual links here - will be a nicer user experience

Reply:
Good point! Will add individual links to the examples